src/cmd/run.js

```diff
@@ -62,7 +62,7 @@ export function defaultWatcherCreator(
 
 export type ReloadStrategyParams = {
   addonId: string,
-  firefox: FirefoxProcess,
+  firefoxApp: FirefoxProcess,
```

Comment:
@shubheksha It is probably better to rename this to `firefoxProcess` (because it is of type [`FirefoxProcess`](https://github.com/mozilla/web-ext/blob/18f38c802ab8df05bb04df7ce504463042170687/src/firefox/index.js#L91), which is a type alias of [`ChildProcess`](https://nodejs.org/api/child_process.html#child_process_class_childprocess)), because it is not the same type of the other properties that you are rename to `firefoxApp` in this patch (in this patch most of the times it is associated to the helper module defined as ["src/firefox/index.js"](https://github.com/mozilla/web-ext/blob/18f38c802ab8df05bb04df7ce504463042170687/src/cmd/run.js#L2), which is a collection of helper functions used to create/copy a Firefox profile, install an extension into the profile, run a firefox instance etc.) and it is going to be helpful to avoid confusion in the future.